src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -733,4 +748,32 @@ export default withApi(
     withSavedSearches(withOrganization(withIssueTags(withProfiler(IssueListOverview))))
   )
 );
+
 export {IssueListOverview};
+
+const StreamRow = styled('div')`
+  display: flex;
+  flex-direction: row;
+`;
+
+const StreamContent = styled('div')<{showSidebar: boolean}>`
+  width: ${p => (p.showSidebar ? '75%' : '100%')};
```

Comment:
Not something for this PR, but it might be nice to have this sidebar float on top of the stream so that it doesn't need to be resized?

Reply:
I'm very into that idea, but we keep saying "we're going to get rid of the sidebar". We'll see